packages/cli/src/start/server.ts

```diff
@@ -0,0 +1,199 @@
+import fs from 'node:fs'
+import path from 'node:path'
+
+import { type AnyModels, handleGetRequest } from '@rise-tools/server'
+import chokidar from 'chokidar'
+import Fastify from 'fastify'
+import internalIp from 'internal-ip'
+import qrcode from 'qrcode-terminal'
+import { WebSocketServer } from 'ws'
+import z from 'zod'
+
+// TODO
+import { connectWebSocket, createWSServerContext } from '../../../server/src/ws-connection'
+import { IGNORED_PATH, WATCH_PATH } from '../config/constants'
+
+interface IndexedModels {
+  [key: string]: any
+}
+
+export const devArgsSchema = z.object({
+  ws: z.boolean().optional().default(false),
+  prod: z.boolean().optional().default(false),
+  port: z.number(),
+  host: z.string(),
+  cwd: z.string().optional(),
```

Comment:
BTW, now that we have this nice Zod schema, it would be great (as a follow-up) to automatically validate passed options and throw an error if their values are missing. There's also nice feature of Zod `describe` which would allow us to attach description to each property.